docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -211,20 +220,32 @@ Variations on the above intervention increased support time or added meal replac
 5.2.1 Population description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  - cohort type
-  - cohort length
-  - age and sex structure
-  - time step
-  - fertility
-  - stratifications 
+  - Cohort type: 
+  	- Prospective open cohort of adults aged 25 and above. New agents should age into the population at age_start
+  	- Size of largest starting population: 100,000 simulants
+  - Cohort length:
+  	- year_start: January 1, 2019
+  	- year_end: December 31, 2040
+  - Age and sex structure:
+  	- Sex: male/female/both
+  	- Age range: age_start=25, age_end=125
+  - Time step:
+  	- One month
+  - Fertility:
+  	- Not applicable
+  - Stratifications:
```

Comment:
I don't understand this section

Reply:
I think it refers to simulants at different disease states/health conditions.